### DISCUSSION TITLE
postgresql jasonb ->> operator

### DISCUSSION THREAD

**Discussion by aathan** [sqlalchemy/sqlalchemy#7763] [Usage Questions] [answered]
Does SQLA support the ->> operator for postgres? How would I look up that answer for myself? I've had trouble coming up with search terms that don't take me down unrelated paths. Thanks in advance!

**Comment by CaselIT** [comment, marked as answer]
Hi,

you could start by searching json or jsonb, that would have brought you to https://docs.sqlalchemy.org/en/14/dialects/postgresql.html#sqlalchemy.dialects.postgresql.JSON that has the documentation you seek

**Reply by aathan** [reply]
Thank you! I guess I had too many other words in my search lol.

**Reply by CaselIT** [reply]
The sphinx search is not the best, so it's probably best to just use one or two words.
